DigitalOffice/Services/UserService/Startup.cs

```diff
@@ -25,6 +32,15 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddControllers();
+
+            CreateTransientObjects(services);
+        }
+
+        private void CreateTransientObjects(IServiceCollection services)
```

Comment:
Not really good, create separate methods for each type of objects e.g. repositories, commands ....

Reply:
fixed